src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -85,7 +85,8 @@ def main(verbose: int, quiet: bool):
 def run(input: str, config: str, rules: str, rejects: str, output: str):
     # Implemented `meta` param in `lexical_index_to_sssom`
 
-    meta = get_metadata_and_prefix_map(config)
+    #meta = get_metadata_and_prefix_map(config)
+    meta = None
```

Comment:
## `lexical_index_to_sssom(meta=None)`
Nico [wrote](https://github.com/monarch-initiative/mondo-ingest/pull/397#issue-2094799963):
> skips `meta` in `src/scripts/match-mondo-sources-all-lexical.py` which is not urgently needed and was causing all the problems @joeflack4 is fixing

I don't know if there would be any negative side effects to this, but you are more familiar with all of the lexmatch outputs and you say that running it locally was fine, so I see no problem w/ that.

Reply:
_[Trish wrote](https://github.com/monarch-initiative/mondo-ingest/pull/397#issuecomment-1906442109)_:
> @joeflack4 have you also run the entire pipeline locally? Did you find that Nico's solution to "ditch meta" worked as intended?

I haven't run the whole pipeline locally. I'll give it a go but I'm terrified of it; I've heard it takes over 24 hours and sometimes this can interfere w/ other projects I'm working on. My computer wasn't actually capable of doing this until last month (without some skipping of things like NCIT). When I've asked Nico if he wants me to do this before, he's said I could hold off. But I'll at least give it a go.

Regarding checking myself the results of `meta=None`, I'll give that a go. I was trusting Nico and it sound like he got the results he expected, but good to have second eyes especially if you think so. Another difference I just noticed as well is that I've also been passing no `meta` (same as passing `None`), but I've been passing `prefix_map` instead. I'm not sure it matters in this case but I'd better remove that as well and run in a more closely to what Nico is running. I'd check out his PR as well but I think better to run on my PR because Nico made some other changes to `prefixes.csv`, but I want to more closely compare the outputs I got before/after with simply the `meta=None`/`prefix_map=None` change.